src/components/FormStepper.js

```diff
@@ -0,0 +1,151 @@
+              primary={true}
+              type="submit"
+              className="button"
+            />
+          </Box>
+        </Form>
+      </Box>
+    </Box>
+  )
+}
+
+const createStepRoute = (basePath, initialState, t, onComplete, onBack, formStepChild) => {
+  const { stepPath, component, mapFormStateToFields, mapFieldsToFormState } = formStepChild.props
+  return (
+    <Route key={stepPath} path={`${basePath}/${stepPath}`}>
+      <InnerFormStep initialState={initialState} onComplete={onComplete} onBack={onBack} t={t} component={component}
+        mapFormStateToFields={mapFormStateToFields} mapFieldsToFormState={mapFieldsToFormState}/>
+    </Route>
+  )
+}
+
+const getCurrentStep = (baseUrl, pathname, stepPaths) => {
+  for (const [stepNumber, stepPath] of stepPaths.entries()) {
+    if (pathname.startsWith(`${baseUrl}/${stepPath}`)) {
+      return {currentStepPath: stepPath, currentStepNumber: stepNumber}
+    }
+  }
+  return {}
+}
+
+const getAdjacentStepUrls = (baseUrl, currentStepPath, stepPaths) => {
+  let previousStepPath = undefined
+  for (const [stepNumber, stepPath] of stepPaths.entries()) {
+    let nextStepPath = stepNumber + 1 >= stepPaths.length ? undefined : stepPaths[stepNumber + 1]
+    if (stepPath === currentStepPath) {
+      return {
+        previousStepUrl: previousStepPath ? `${baseUrl}/${previousStepPath}` : undefined,
+        nextStepUrl: nextStepPath ? `${baseUrl}/${nextStepPath}` : undefined
+      }
+    }
+    previousStepPath = stepPath
+  }
+  return {}
+}
+
+const FormStepper = ({ initialFormState, onFormComplete, t, children }) => {
+  const { url, path } = useRouteMatch();
+  const [formState, setFormState] = useState(initialFormState);
+  const history = useHistory();
+  const { pathname } = useLocation();
+  const stepPaths = React.Children.map(children, child => child.props.stepPath)
+  const {currentStepPath, currentStepNumber} = getCurrentStep(url, pathname, stepPaths)
+  const {previousStepUrl, nextStepUrl} = getAdjacentStepUrls(url, currentStepPath, stepPaths)
+  const onComplete = nextStepUrl
+    ? (formStateUpdates) => {
+        setFormState(formState => ({...formState, ...formStateUpdates}))
+        history.push(nextStepUrl)
+      }
+    : (formStateUpdates) => {
+        setFormState(formState => ({...formState, ...formStateUpdates}))
+        const finalFormState = {...formState, ...formStateUpdates}
+        onFormComplete(finalFormState)
+      }
+  const onBack = previousStepUrl ? () => history.push(previousStepUrl) : undefined
+  const stepperBarSteps = React.Children.map(children, child => ({
+    title: child.props.title,
+    onClick: () => history.push(`${url}/${child.props.stepPath}`)
+  }))
+  return (
+    <Box tag="section" basis="auto">
+      <Disclaimer />
+      <Heading tag="h1">{t('powerOfAttorney')}</Heading>
+      <div>
+        <div className="stepper">
+          <Stepper
+            steps={stepperBarSteps}
+            activeColor="#679ba1"
```

Comment:
This looks to be `POA_CORAL_BLUE` we can pull it from a theme/styles file.

Reply:
Good catch - I'll make this reference the constant.